reference/security-summary.md

```diff
@@ -10,108 +14,111 @@ the chart's documentation for instructions.
 
 ## Chart security reference
 
-| Chart                               | Pod security policy       |
-|-------------------------------------|---------------------------|
-| ibm-ace-dashboard-dev               | ibm-anyuid-psp            |
-| ibm-ace-server-dev                  | ibm-anyuid-psp            |
-| ibm-ace-dashboard-prod              | ibm-anyuid-psp            |
-| ibm-ace-server-prod                 | ibm-anyuid-psp            |
-| ibm-ace-dev                         | ibm-anyuid-psp            |
-| ibm-ace-prod                        | ibm-anyuid-psp            |
-| ibm-apiconnect-ent                  | ibm-privileged-psp        |
-| ibm-apiconnect-pro                  | ibm-privileged-psp        |
-| ibm-aspera-cli                      | ibm-anyuid-psp            |
-| ibm-blockchain-platform-remote-peer | ibm-anyuid-hostpath-psp   |
-| ibm-calico-bgp-peer                 | ibm-privileged-psp        |
-| ibm-cam                             | ibm-anyuid-hostpath-psp   |
-| ibm-cem                             | ibm-anyuid-psp            |
-| ibm-cloud-appmgmt-prod              | ibm-privileged-psp        |
-| ibm-cloud-mgmt-platform-prod        | ibm-privileged-psp        |
-| ibm-csi-nfs                         | ibm-privileged-psp        |
-| ibm-datapower-dev                   | ibm-anyuid-psp            |
-| ibm-datapower-nonprod               | ibm-anyuid-psp            |
-| ibm-datapower-prod                  | ibm-anyuid-psp            |
-| ibm-dba-content-prod                | ibm-privileged-psp        |
-| ibm-dba-mutlicloud-prod             | ibm-privileged-psp        |
```

Comment:
This is the only difference which I saw, this is not in the CSV which I used as the base.
```
cloud@idpx34:~/cloud-pak(master)$ egrep -v -f new old 
ibm-dba-mutlicloud-prod
```
```
cloud@idpx34:~/cloud-pak(master)$ egrep -v -f old new 
```

Reply:
I could not find the chart in the public or private repos, I assume that it has been removed.